demos/common/cpp/models/include/models/detection_model_yolo.h

```diff
@@ -63,10 +65,11 @@ class ModelYolo3 : public DetectionModel {
         const unsigned long resized_im_h, const unsigned long resized_im_w, const unsigned long original_im_h,
         const unsigned long original_im_w, std::vector<DetectedObject>& objects);
 
-    static int calculateEntryIndex(int side, int lcoords, int lclasses, int location, int entry);
+    static int calculateEntryIndex(int entriesNum, int lcoords, int lclasses, int location, int entry);
```

Comment:
Well, the `totalCells` is more correct name than `entriesNum` -  this variable represents total "detection cells" in each output layer for YOLO* architectures.

Reply:
Done